Package.swift

```diff
@@ -0,0 +1,19 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let package = Package(
+    name: "XLActionController",
+        platforms: [
+        .iOS(.v9)
+    ],
+    products: [
+        .library(name: "XLActionController", targets: ["XLActionController"])
+    ],
+    targets: [
+        .target(
```

Comment:
I haven't had a deeper look into what SPM supports but would it be possible to support multiple subtargets? (as in Cocoapods there are subspecs)

Reply:
Yeah you can support multiple targets with SPM. I'm no SPM expert but RxSwift are a good example of this [here](https://GitHub.com/reactivex/rxswift/blob/main/Package.swift).

I didn't do it here as I didn't need it to consume this library in my project. Happy to change for this PR though.

Reply:
I think it would be good to be consistent with how the cocoapods subspecs are set up